com.unity.multiplayer.mlapi/Tests/Runtime/NetworkTransformTests.cs

```diff
@@ -0,0 +1,151 @@
+using System;
+using System.Collections;
+using System.Text.RegularExpressions;
+using MLAPI.Prototyping;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using static MLAPI.Prototyping.NetworkTransform;
+
+namespace MLAPI.RuntimeTests
+{
+    [TestFixture(true)]
+    [TestFixture(false)]
+    public class NetworkTransformTests : BaseMultiInstanceTest
```

Comment:
Pretty nifty helper you have there and like the tests side too... interesting indeed!  💯